README.adoc

```diff
@@ -41,14 +41,14 @@ Not to be confused with the credentials the **Auth Server** used to authenticate
 
 == Configuration
 
-The user information and authorities returned by the service can be configured:
+The user information and authorities returned by the service are configured in the service config.  Multiple users can eb configured as shown in the following example.
 
 [source,yaml]
 ----
 smartcosmos:
   security:
-    user:
-      name: user
+    users[user]:
```

Comment:
Isn't it possible to do something like this?

```
users:
    - username: user
      password: password
      ...
    - username: user2
      password: password2
      ...
```

In Ansible (which also uses YML) this syntax works perfectly.